src/App.tsx

```diff
@@ -82,11 +94,12 @@ export const App: React.FC = () => {
                   </div>
                 )}
 
-                {author && loaded && !hasError && posts.length > 0 && (
+                {author
+                && loaded && !hasError && posts.length > 0 && (
```

Comment:
consider moving long ternaries to a separate const with consistent name and use the const here